stats/stats.go

```diff
@@ -60,7 +61,20 @@ func StartStatsServerWithPort(startupOpts StartupOptions, addhandlers ...func(mu
 	}
 
 	var logLevel zap.AtomicLevel
-	if startupOpts.LogLevel != nil {
+	if envLogLevel := os.Getenv("LOG_LEVEL"); envLogLevel != "" {
+		var setLevel zapcore.Level
+		switch envLogLevel {
+		case "debug":
+			setLevel = zapcore.DebugLevel
+		case "warn":
+			setLevel = zapcore.WarnLevel
+		case "error":
+			setLevel = zapcore.ErrorLevel
+		default:
+			setLevel = zapcore.InfoLevel
+		}
```

Comment:
There are also `DPanic`, `Panic`, and `Fatal` levels but I decided to just start with these. Open to thoughts on which levels should be configurable.

Reply:
considering the low cost, I think we should add `Panic` and `Fatal` even though I don't think they'll really be used in practice